crmsh/bootstrap.py

```diff
@@ -2106,11 +2101,20 @@ def bootstrap_init(context):
             utils.fatal("Expected NODE argument to csync2_remote")
         _context.cluster_node = args[1]
 
+    if stage and _context.cluster_is_running and not utils.service_is_active(CSYNC2_SERVICE):
+        _context.skip_csync2 = True
```

Comment:
When running `crm cluster init csync2 -y`, this flag `skip_csync2` will be set as `True`